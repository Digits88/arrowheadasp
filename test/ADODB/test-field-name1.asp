<!--#include file="../config/ADODB.inc"-->
Here is the test of Field Name.<br>
<%
    Dim CN, RS

    Set CN = Server.CreateObject("ADODB.Connection")
    CN.Open Connect_URL, Connect_Username, Connect_Password
    Set RS = CN.Execute("SELECT StringField, IntField FROM Test ORDER BY IntField")

    %>Name("StringField"): <%=RS("StringField").Name%><br>
    Name("IntField"): <%=RS("IntField").Name%><br>
<%

    CN.Close
%>

The End.
