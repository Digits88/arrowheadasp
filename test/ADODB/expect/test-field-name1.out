
Here is the test of Field Name.<br>
Name("StringField"): ^[Ss]tring[Ff]ield$<br>
    Name("IntField"): ^[Ii]nt[Ff]ield$<br>


The End.
